todo/repositories/team_repository.py

```
@@ -0,0 +1,79 @@
+from datetime import datetime, timezone
+from typing import Optional
+from bson import ObjectId
+
+from todo.models.team import TeamModel, UserTeamDetailsModel
+from todo.repositories.common.mongo_repository import MongoRepository
+from todo.models.common.pyobjectid import PyObjectId
```

Ruff (0.11.9)

`todo.models.common.pyobjectid.PyObjectId` imported but unused

Remove unused import: `todo.models.common.pyobjectid.PyObjectId`

(F401)